.github/workflows/nightly.yaml

```
@@ -84,3 +129,9 @@ jobs:
           OPERATOR_MANIFEST: ${{ env.OPERATOR_MANIFEST }}
           IMG: ${{ env.OPERATOR_IMAGE }}
         run: make test-e2e
+
+      - name: Remove SeaLights secrets
+        if: ${{ matrix.branch == 'main' }}
```

```suggestion
        if: always() && matrix.branch == 'main'
```

Similar to my comment in https://github.com/redhat-developer/rhdh-operator/pull/1107#discussion_r2090903084